nex/secure-connection/register_ex.go

```diff
@@ -40,10 +40,10 @@ func RegisterEx(err error, client *nex.Client, callID uint32, stationUrls []*nex
 
 	localStationURL := localStation.EncodeToString()
 
-	rmcResponseStream := nex.NewStreamOut(globals.NEXServer)
+	rmcResponseStream := nex.NewStreamOut(globals.SecureServer)
 
 	rmcResponseStream.WriteUInt32LE(0x10001) // Success

```

Comment:
Since we are making changes on friends, can we change this here?

```suggestion
	retval := nex.NewResultSuccess(nex.Errors.Core.Unknown)
	
	rmcResponseStream.WriteResult(retval)
```

Reply:
Change applied

Reply:
If you also want to get technical, this isn't a uint32 type either. It's a Result type. This server was made before the stream structs had support for Result types so they were treated as uint32s. It's the same value (a Result just stores a uint32) but semantically this is wrong